addon-test-support/-private/helpers.js

```diff
@@ -320,3 +321,46 @@ export function getProperty(object, pathToProp) {
 
   return typeof value === 'function' ? value.bind(propOwner) : value;
 }
+
+// recursively convert any page object child property to its definition
+
+// since page object properties are mostly getter based, page objects retain
+// a copy of their pojo definition representation to facilitate composition 
+// (since accessing the page object's properties during creation here
+// would fire off the getters).
+export function convertPageObjectPropsToDefinitions(definition){
```

Comment:
I think we should avoid one more recursive mechanism. We already have `ceibo` for that.